content/de/guides/_index.md

```diff
@@ -130,7 +130,7 @@ toc_type = "single"
 ### Getting Started
 
 - Installing Tools ([Windows](/guides/rhinocommon/installing-tools-windows/), [Mac](/guides/rhinocommon/installing-tools-mac/))
-- Your First Plugin ([Windows](/guides/rhinocommon/your-first-plugin-windows/), [Mac](/guides/rhinocommon/your-first-plugin-mac/), [Cross-Platform](/guides/rhinocommon/your-first-plugin-crossplatform/))
+- Your First Plugin ([Windows](/guides/rhinocommon/your-first-plugin-windows/), [Mac](/guides/rhinocommon/your-first-plugin-mac/)
```

Comment:
I don't know if this change was intended, but this is not the english guide so probably shouldn't be changed.

Reply:
Great catch!